src/Language/PureScript/Errors.hs

```diff
@@ -410,37 +401,37 @@ prettyPrintSingleError full level e = prettyPrintErrorMessage . positionHintsFir
             , prettyPrintParseError err
             ]
     renderSimpleErrorMessage (ErrorParsingFFIModule path) =
-      paras [ line "Unable to parse module from FFI file: "
+      paras [ line "Unable to parse foreign module:"
             , indent . line $ path
+            , line "Ensure that all foreign modules contain a single module header at the start of a line, as follows:"
+            , indent $ line "// module A.B.C"
             ]
     renderSimpleErrorMessage (ErrorParsingModule err) =
       paras [ line "Unable to parse module: "
             , prettyPrintParseError err
             ]
     renderSimpleErrorMessage (MissingFFIModule mn) =
-      line $ "Missing FFI implementations for module " ++ runModuleName mn
+      paras [ line $ "The foreign module implementation for module " ++ runModuleName mn ++ " is missing."
+            , line $ "Provide your foreign module implementation using the --ffi command line option, and ensure that your module contains the following module header:"
```

Comment:
A minor issue - errors are generic (shared across any potential backend) but with these messages the FFI errors become specific to the PureScript backend now.